tensorboard/data/server/data_compat.rs

```diff
@@ -286,6 +287,50 @@ impl SummaryValue {
     }
 }
 
+fn tf1x_image_metadata() -> Box<pb::SummaryMetadata> {
+    let plugin_content = pb::ImagePluginData {
```

Comment:
Just wondering: in principle we could just construct the proto and serialize it once, and just copy the serialized bytes into each metadata, right?  I guess it's probably not worth it given that this is only initial metadata so we don't have to do this on every event.

Reply:
Indeed. You have to worry a bit about how/when that’s constructed and
accessed if you do that. The whole thing is easy if your computation can
be `const` (C++ `constexpr`), but anything involving allocation won’t be
(yet). More generally, Rust is picky about shared mutable globals—you
know, memory safety and all. [`once_cell::sync::OnceCell`] is a pretty
standard solution here, and ergonomic enough. But given that, as you
note, this is not on any critical path, I’d prefer to keep it simple.

[`once_cell::sync::OnceCell`]: https://docs.rs/once_cell/1.4.0/once_cell/sync/struct.OnceCell.html